tests/ci/create-commit.sh

```diff
@@ -103,267 +103,15 @@ build_image() {
 ##
 ##################################################
 
-# Write a source for the Neptune app.
-tee "$SOURCE_FILE" > /dev/null << EOF
-id = "copr_neptune"
-name = "copr_neptune"
-type = "yum-baseurl"
-url = "https://download.copr.fedorainfracloud.org/results/pingou/qtappmanager-fedora/epel-8-${ARCH}"
-check_gpg = false
-check_ssl = false
-system = false
-EOF
+# Set the correct arch for the the Neptune app source file.
+sed -i "s|@@ARCH@@|${ARCH}|g" "$SOURCE_FILE"
```

Comment:
what about having the files as templates and use envsubst instead?

https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html

Reply:
I'm use to this kind of templates, I haven't used `envsubst` before, I'll give it a try.
Thanks.